sanic/config.py

```diff
@@ -201,4 +201,7 @@ def load_environment_vars(self):
         for k, v in os.environ.items():
             if k.startswith(SANIC_PREFIX):
                 _, config_key = k.split(SANIC_PREFIX, 1)
-                self[config_key] = v
+                if v.isdigit():
+                    self[config_key] = int(v)
```

Comment:
@seemethere I was thinking that too at first but we don't have any float values in the config right now... I think we can use float instead of int though and the timeout should be happy.